api/src/main/java/org/openmrs/module/patientflags/db/FlagDAO.java

```diff
@@ -92,6 +108,14 @@
 	 * @throws DAOException
 	 */
 	public Tag getTag(String name) throws DAOException;

```

Comment:
You need to have at least a line between methods.

Reply:
Done